css/index.styl

```diff
@@ -1210,6 +1210,7 @@ loadingSize = 30px
 .auth0-lock-password-strength
   width 100%
   bottom 41px
+  left 0
```

Comment:
Now the password strength field is position relative to the password input, we have to add left 0 to take the padding for the icon into account (other wise the password strength tooltip would be 40px to the right)